pyznap/ssh.py

```diff
@@ -84,14 +84,7 @@ def __init__(self, user, host, key=None, port=22, compress=None, **kwargs):
 
         self.cmd = ['ssh', '-i', self.key, '-o', 'ControlMaster=auto', '-o', 'ControlPersist=1m',
                     '-o', 'ControlPath={:s}'.format(self.socket), '-p', str(self.port),
-                    '{:s}@{:s}'.format(self.user, self.host)]
-
-        if kwargs.get("ServerAliveInterval"):
-            self.cmd.insert(1, '-o')
-            self.cmd.insert(2, 'ServerAliveInterval={}'.format(kwargs.get('ServerAliveInterval')))
-        if kwargs.get("ServerAliveCountMax"):
-            self.cmd.insert(1, '-o')
-            self.cmd.insert(2, 'ServerAliveCountMax={}'.format(kwargs.get('ServerAliveCountMax')))
+                    '-o', 'ServerAliveInterval=60', '{:s}@{:s}'.format(self.user, self.host)]
```

Comment:
can we add the `ServerAliveInterval` and `ServerAliveCountMax` config back some where?
I think 60s is too long sometimes (e.g., when debuging the retry feature, i want 10s)

Reply:
Atm I feel strongly towards not adding too many options. I've changed the default to a more reasonable 30s, and for debugging you will have the code open anyway and can quickly set it to something small.